kibana/Dockerfile

```diff
@@ -26,7 +26,13 @@ LABEL io.k8s.description="Kibana container for querying Elasticsearch for aggreg
       version="3.6.0" \
       release="1"
 
-RUN yum-config-manager --enable rhel-7-server-ose-3.6-rpms && \
+USER 0
+
+RUN yum repolist > /dev/null && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
```

Comment:
What packages are we still pulling from these two repos, that are not already baked into the base image?